aeppl/abstract.py

```diff
@@ -69,15 +69,19 @@ def assign_custom_measurable_outputs(
 
     new_node = node.clone()
     op_type = type(new_node.op)
-    new_op_type = type(
-        f"{type_prefix}{op_type.__name__}", (op_type,), op_type.__dict__.copy()
-    )
-
-    new_node.op = copy(new_node.op)
-    new_node.op.__class__ = new_op_type
+    op_name = op_type.__name__
 
     # TODO: The above could be a stand-alone utility function for all sorts of
     # instance-based dispatching
+    if (
```

Comment:
We shouldn't use names for anything functional in Aesara-based code.

Reply:
How should we go about checking if an Op has already been dispatched a measurable_outputs_fn?